Fix arduino installer

bil-elmoussaoui opened the pull request.

satya164 commented:
Having the version name in the installed files will make uninstall hard. Can we remove th version number from installed files?

bil-elmoussaoui commented:
Done, I tried the script on my desktop and everything works fine.
I removed this line
`-cp "/opt/arduino/arduino.desktop" "/usr/local/share/applications"`
As the desktop file is being created and moved by the `install.sh` file.
